base_multi_company/models/company_activate.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class CompanyActivate(models.Model):
+
+    _name = 'company.activate'
+    _description = 'Company Activations'
+
+    is_active = fields.Boolean(
```

Comment:
You're right, but it can be a possible side effect, as I have found that some triggers are launched with sudo rights, making this not work for properties. Maybe this is not happening in 10.0. It worths to be checked.